test/hardhat/integration/xrp-genesis.test.js

```
@@ -0,0 +1,395 @@
+        expect(prevPower).to.be.at.least(currPower);
+      }
+    });
+
+    it('should generate deterministic output', async function() {
+      const mockStakes = createMockXRPStakes();
+
+      const genesisState1 = await generateXRPGenesisState([...mockStakes]);
+      const genesisState2 = await generateXRPGenesisState([...mockStakes]);
+
+      // Remove timestamp fields for comparison
+      delete genesisState1.genesis_time;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)